### ISSUE TITLE
MasterPDF loosing all initials after restart

### ISSUE THREAD

**Issue by sonicnkt** [flathub/net.codeindustry.MasterPDFEditor#65]
I noticed that when i completely close MasterPDF it looses all initials i created previously:
https://code-industry.net/masterpdfeditor-help/inserting-initials/

This works fine on windows and the native linux releases on their homepage.

Initials (and custom stamps) will be stored in `~/.masterpdfeditor/` in the official linux version. Maybe this directory is not configured correctly in this flatpak?

**Comment by muelli** [comment]
yeah, we're not exposing that directory: https://github.com/flathub/net.codeindustry.MasterPDFEditor/blob/86538d7727264b54b8cd745f27e1abf13afe9047/net.codeindustry.MasterPDFEditor.yaml#L6

Hm. Ideally, the app would use the XDG directories.

I see two ways to make it work:
1) We could probably symlink ~/.masterpdfeditor to $XDG_CONFIG_HOME
2) add ~/.masterpdfeditor to the manifest.

**Comment by muelli** [comment]
https://github.com/flathub/net.codeindustry.MasterPDFEditor/issues/19 seems related, somehow.
